data-protocols/ids/ids-api-multipart-dispatcher-v1/src/main/java/org/eclipse/dataspaceconnector/ids/api/multipart/dispatcher/sender/MultipartArtifactRequestSender.java

```diff
@@ -110,8 +110,8 @@ protected Message buildMessageHeader(DataRequest request, DynamicAttributeToken
                 ._transferContract_(contractId)
                 .build();
 
-        var path = idsApiPath + (idsApiPath.endsWith("/") ? "data" : "/data");
-        message.setProperty(IDS_WEBHOOK_ADDRESS_PROPERTY, idsWebhookAddress + path);
+        var webhookPath = idsApiPath + (idsApiPath.endsWith("/") ? "data" : "/data");
```

Comment:
Couldn't this composition be done on `IdsMultipartDispatcherServiceExtension` so it will be done just one time and only one complete url parameter will be passed to the senders?

Reply:
Agreed, I'll refactor it.